monorepo-bench/Makefile

```diff
@@ -0,0 +1,14 @@
+# Intended to be used from inside docker containers built from monorepo-bench.Dockerfile
+RUNNER = _build/default/bench.exe
+DUNE_TO_BENCHMARK = /home/user/dune/_build/default/bin/main.exe
+
+$(RUNNER): dune bench.ml
+	dune build $@ --release
+
+bench: $(RUNNER)
```

Comment:
Shouldn't this depend on `$(DUNE_TO_BENCHMARK)` as well?

Reply:
I don't think so. `$(DUNE_TO_BENCHMARK)` isn't a target of this makefile; it's assumed that it's already been built when `make bench` is run.